src/pat/tinymce/templates/upload.xml

```diff
@@ -1,7 +1,7 @@
 <div>
-  <div class="linkModal">
+  <div class="linkModal22">
     <h1><%- uploadHeading %></h1>
-    <input type="text" name="upload" style="display:none" />
-    <div class="uploadify-me"></div>
+    <input type="text" name="upload22" style="display:none" />
+    <div class="uploadify-me22"></div>
```

Comment:
Why do the classes in here have the suffix `22`?

Reply:
@MrTango was this intentionally or a mistake? see https://github.com/plone/mockup/commit/93d2876114f4feb2f6a340b8003f3fbb7c5716f3#diff-34976cb7c710995cf0a4bf63c313426e4921ec7386d82e9c42f85fd489a65a66R2